app/services/facets/entry.rb

```diff
@@ -0,0 +1,67 @@
+module Facets
+  class Entry
+    attr_accessor :name, :model, :helper, :extension,
```

Comment:
I wanted to create public readers, private writers pattern here. I'll fix it to `attr_reader` instead.
I agree that good documentation needed here, but not everybody reads it. I prefer adding at least basic integrity check here, since this is the place where the error actually is. I want to minimize the cases when the user get an error at some random point in code (let's say in API controller) that doesn't tell the user too much.